source/beamline/VESPERS/VESPERSEndstation.h

```diff
@@ -111,7 +114,7 @@ public slots:
 	/// Sets the current control that should be focused on.  It will find the correct control based on the name.  Control is set to 0 if invalid name is given.
 	void setCurrent(QString name);
 	/// Toggles the current state of the laser power.
-	void toggleLaserPower() { toggleControl(laserPower_); emit laserPoweredChanged(); }
+	void toggleLaserPower() { laserPower_->move(1); emit laserPoweredChanged(); }
```

Comment:
No, thank god VESPERS is the only one to have a PV state machine in this way.  If it was I'd agree, but since I've only seen it on VESPERS I'm okay leaving it as a VESPERS specific control.